api/src/main/java/org/pmiops/workbench/elasticsearch/ElasticDocument.java

```diff
@@ -62,6 +63,7 @@ String lower() {
                   .put("visit_concept_id", esType(ElasticType.KEYWORD))
                   .put("value_as_number", esType(ElasticType.FLOAT))
                   .put("value_as_concept_id", esType(ElasticType.KEYWORD))
+                  .put("value_source_concept_id", esType(ElasticType.KEYWORD))
```

Comment:
nit: should this be `value_as_source_concept_id` to match the other naming?

Reply:
Renamed to value_as_source_concept_id and also added a test case to cover this case.